keylime/ima/ima.py

```diff
@@ -472,7 +492,53 @@ def read_allowlist(al_path=None, checksum="", gpg_sig_file=None, gpg_key_file=No
                 f"Checksum of allowlist does not match! Expected {checksum}, Calculated {calculated_checksum}"
             )
 
+    # Return artifacts as a JSON bundle, ready for sending
+    bundle = {
+        "ima_policy": base64.b64encode(alist_bytes).decode(),
+        "key": base64.b64encode(al_key).decode(),
+        "sig": base64.b64encode(al_sig).decode(),
+        "checksum": checksum,
+    }
+
+    return bundle
+
+
+class SignatureValidationError(Exception):
+    def __init__(self, message, code):
+        self.message = message
+        self.code = code
+        super().__init__(self.message)
+
+
+# Reads allowlist bundles sent to the verifier, validates them, and processes them. Returns an allowlist.
+def unbundle_ima_policy(allowlist_bundle, force_verification=False):
```

Comment:
```suggestion
def unbundle_ima_policy(allowlist_bundle, force_verification=True):
```
I think this function should be secure by default

Reply:
@THS-on Ahhh, I understand what you were asking for now. I agree, that makes sense.

I've renamed the option to `verify` from `force_verification` - I think that is more consistent with how it's supposed to be used now - and moved the config check to the verifier code.

And thank you so much for your reviews! They have been very helpful in getting this ready 😄